src/frequenz/sdk/actor/power_distributing/result.py

```diff
@@ -84,9 +84,16 @@ class PowerBounds:
     """Inclusion and exclusion power bounds for requested batteries."""
 
     inclusion_lower: float
+    """The inclusion power lower bound for the requested batteries."""
```

Comment:
This sounds a bit funny, but I couldn't come up with a phrasing that is more clear, but mentioning it in case someone can think of something better 😁

Reply:
@shsms do you have any suggestion here?

Reply:
5 options:

The lower {value/end/limit} of the inclusion power bounds ...
The lower bound of the {range/span} where power requests will be accepted ...

Reply:
that's how they're described in the battery pool,  and in the microgrid api: https://github.com/frequenz-floss/frequenz-sdk-python/blob/a8f3d62e93d226278c71fc650033fd00758d62a1/src/frequenz/sdk/timeseries/battery_pool/_result_types.py#L35

Reply:
Exactly. Here it's just the internal representation of the power bounds so it makes sense to me to only explain it in terms of bounds/ranges rather than the inclusion/exclusion concept